content/en/agent/cluster_agent/admission_controller.md

```diff
@@ -63,12 +63,72 @@ To enable the admission controller for the Datadog operator, set the parameter `
 [...]
 ```
 
+### Manual setup
+
+To enable the admission controller without using Helm or the Datadog operator, you'll need to add a few things to your configuration:
+
+First, Download the [Cluster Agent RBAC permissions][3] manifest, and add the following under `rules`:
+
+{{< code-block lang="yaml" filename="cluster-agent-rbac.yaml" disable_copy="true" >}}
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - mutatingwebhookconfigurations
+  verbs: ["get", "list", "watch", "update", "create"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "list", "watch", "update", "create"]
+- apiGroups: ["batch"]
+  resources: ["jobs", "cronjobs"]
+  verbs: ["get"]
+- apiGroups: ["apps"]
+  resources: ["statefulsets", "replicasets", "deployments"]
+  verbs: ["get"]
+{{< /code-block >}}
+
+Add the following to the bottom of `agent-services.yaml`:
+
+{{< code-block lang="yaml" filename="agent-services.yaml" disable_copy="true" >}}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: datadog-cluster-agent-admission-controller
+  labels:
+    app: "datadog"
+    app.kubernetes.io/name: "datadog"
+spec:
+  selector:
+    app: datadog-cluster-agent
+  ports:
+  - port: 443
+    targetPort: 8000
+
+{{< /code-block >}}
+
+Add environment variables to the Cluster Agent deployment which enable the Admission Controller:
+
+{{< code-block lang="yaml" filename="cluster-agent-deployment.yaml" disable_copy="true" >}}
+- name: DD_ADMISSION_CONTROLLER_ENABLED
+  value: "true"
+- name: DD_ADMISSION_CONTROLLER_MUTATE_UNLABELLED
+  value: "false"
```

Comment:
this is a an optional config, and it's `false` by default - should we add it as a note?

Reply:
Ok, how does this look:

```yaml
- name: DD_ADMISSION_CONTROLLER_ENABLED
  value: "true"
- name: DD_ADMISSION_CONTROLLER_SERVICE_NAME
  value: "datadog-cluster-agent-admission-controller"

# Uncomment this to configure APM tracers automatically (see below)
# - name: DD_ADMISSION_CONTROLLER_MUTATE_UNLABELLED
#   value: "true"
```

Reply:
looks good thanks - to be validated by docs cc @cswatt